.github/workflows/check-coverage-deploy.yaml

```diff
@@ -54,29 +115,36 @@ jobs:
           cd ${PKG_NAME}
           R CMD build . --md5
 
-      - run: ls
-      - run: ls ${PKG_NAME}
-      - run: ls drat
       - run: echo ${{ github.ref }}
+
+      - name: Set environmental variables
+        run: |
+          setwd('${PKG_NAME}')
```

Comment:
Are you sure this line is going to work in R?

Reply:
I was pretty sure, but I'm testing now.

Reply:
It wasn't, but I was able to use the working directory to change it!